tests/cases/dom/manipulate/createElement.js

```diff
@@ -0,0 +1,157 @@
+import QUnit from "qunitjs";
+import {createElement} from "../../../../dom/manipulate";
+
+QUnit.module("dom/manipulate/createElement()",
+    {
+        beforeEach: () =>
+        {
+            document.getElementById("qunit-fixture").innerHTML = "";
+        },
+    }
+);
```

Comment:
I used the beforeEach to clean the fixure element after every test I made. This is because of the fact that I added my created element to the "qunit-fixure" element on all non-error-throwing tests. But because of my new tests I don't need a empty element to insert something anymore.